ci/bump.sh

```diff
@@ -65,5 +65,9 @@ function msg {
 msg "Updating Cargo manifests with new version '$version':"
 for manifest in ethcontract*/Cargo.toml; do
 	msg "  - $manifest"
-	sed -i -E -e 's/^((ethcontract-[a-z]+ = \{ )?version) = "[0-9\.]+"/\1 = "'"$version"'"/g' "$manifest"
+	if [[ $(uname) == Darwin ]]; then
+		sed -i '' -E -e 's/^((ethcontract(-[a-z]+)? = \{ )?version) = "[0-9\.]+"/\1 = "'"$version"'"/g' "$manifest"
```

Comment:
Also notice that this regex was changed to update `ethcontract = ...` as well (previously it was `ethcontract-... = ...`). I totally forgot about this change when creating this PR.